src/styles/sharedComponents/buttons/buttonVariants.js

```diff
@@ -0,0 +1,51 @@
+// @flow strict-local
+
+import { StyleSheet } from "react-native";
+import type { TextStyleProp, ViewStyleProp } from "react-native/Libraries/StyleSheet/StyleSheet";
+
+import colors from "../../colors";
+
+const PRIMARY = "#5D8017";
+const WARNING = "#9B1111";
+const NEUTRAL = "#979797";
+
+const viewStyles: { [string]: ViewStyleProp } = StyleSheet.create( {
+  containerDefault: {
+    alignItems: "center",
+    paddingVertical: 10,
+    paddingHorizontal: 10,
+    borderRadius: 40,
+    marginVertical: 15,
+    marginHorizontal: 10
+  },
+  containerPrimary: {
+    backgroundColor: PRIMARY,
+    borderColor: PRIMARY
+  },
+  containerWarning: {
+    backgroundColor: WARNING,
+    borderColor: WARNING
+  },
+  containerNeutral: {
+    backgroundColor: NEUTRAL,
+    borderColor: NEUTRAL
+  },
+  containerDisabled: {
+    opacity: 0.65
```

Comment:
I know this way is simple, but eventually you'll get into a situation where a button overlays something else and the background element will be partially visible, which is almost never what you want, so please explicitly style the disabled states of the different levels as they are in Figma:

Primary: `#C6DC98`
Neutral: `#D3D3D3`
Warning: [theoretically never disabled, but I would put something in here anyway, maybe just grab the hex code for whatever `#9B1111` is at 0.65 opacity.]